apps/wallet/src/ui/app/components/transactions-card/index.tsx

```diff
@@ -48,42 +49,15 @@ export function TransactionCard({
     txn: SuiTransactionBlockResponse;
     address: SuiAddress;
 }) {
-    const transaction = getTransactionKind(txn)!;
     const executionStatus = getExecutionStatusType(txn);
-    getTransactionKindName(transaction);
 
     // const objectId = useMemo(() => {
     //     return getTxnEffectsEventID(txn.events!, address)[0];
     // }, [address, txn.events]);
 
-    const transfer = useGetTransferAmount({
-        txn,
-        activeAddress: address,
-    });
-
-    // we only show Sui Transfer amount or the first non-Sui transfer amount
-    const transferAmount = useMemo(() => {
-        // Find SUI transfer amount
-        const amountTransfersSui = transfer?.find(
-            ({ coinType }) => coinType === SUI_TYPE_ARG
-        );
-
-        // Find non-SUI transfer amount
-        const amountTransfersNonSui = transfer?.find(
-            ({ coinType }) => coinType !== SUI_TYPE_ARG
-        );
-
-        return {
-            amount:
-                amountTransfersSui?.amount ||
-                amountTransfersNonSui?.amount ||
-                null,
-            coinType:
-                amountTransfersSui?.coinType ||
-                amountTransfersNonSui?.coinType ||
-                null,
-        };
-    }, [transfer]);
+    const { amount, coinType } = useGetTransferAmount(txn, address);
+
+    const [, symbol] = useFormatCoin(amount, coinType);
```

Comment:
To avoid using the `useFormatCoin` hook I think there is a method that extracts the symbol from coin type `Coin.getCoinSymbol`

Reply:
thanks for that, updated